### PR TITLE
add documentation II

### PR DESCRIPTION
**Pull request by h-mayorquin**
Last commmit of the previous PR was lost.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ pip install -U ndx-binned-spikes
 
 ## Usage
 
+The `BinnedAlignedSpikes` object is designed to store counts of spikes around a set of events (e.g. stimuli or behavioral events such as licks). The events are characterized by their timestamps and a bin data structure is used to store the spike events around each of the event timestamps. The `BinnedAlignedSpikes` keeps a separate count for each of the units (e.g. neurons).
```
Comment (CodyCBakerPhD) [inline README.md]:
```suggestion
The `BinnedAlignedSpikes` object is designed to store counts of spikes around a set of events (e.g. stimuli or behavioral events such as licks). The events are characterized by their timestamps and a bin data structure is used to store the spike events around each of the event timestamps. The `BinnedAlignedSpikes` keeps a separate count for each of the units (e.g., neurons). 
```

File: README.md [hunk 2]
```diff
@@ -62,24 +64,26 @@ ecephys_processinng_module.add(binned_aligned_spikes)
 ```
 
 ### Parameters and data structure
+The structure of the bins are characterized with the following parameters:
+ 
+* `milliseconds_from_event_to_first_bin`: The time in milliseconds from the event to the first bin. A negative values indicates that the first bin is before the event whereas a positive value indicates that the first bin is after the event.
```
Comment (CodyCBakerPhD) [inline README.md]:
```suggestion
* `milliseconds_from_event_to_first_bin`: The time in milliseconds from the event to the first bin. A negative value indicates that the first bin is before the event whereas a positive value indicates that the first bin is after the event. 
```

**Review by CodyCBakerPhD — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -11,6 +11,8 @@ pip install -U ndx-binned-spikes
 
 ## Usage
 
+The `BinnedAlignedSpikes` object is designed to store counts of spikes around a set of events (e.g. stimuli or behavioral events such as licks). The events are characterized by their timestamps and a bin data structure is used to store the spike counts around each of the event timestamps. The `BinnedAlignedSpikes` object keeps a separate count for each of the units (e.g. neurons), in other words, the the spikes of the units are counted separately but aligned to the same set of events.
```
Comment (CodyCBakerPhD) [inline README.md]:
```suggestion
The `BinnedAlignedSpikes` object is designed to store counts of spikes around a set of events (e.g., stimuli or behavioral events such as licks). The events are characterized by their timestamps and a bin data structure is used to store the spike counts around each of the event timestamps. The `BinnedAlignedSpikes` object keeps a separate count for each of the units (e.g., neurons), in other words, the spikes of the units are counted separately but aligned to the same set of events.
```

Noticed you had to force push - let me know if this still isn't up to date (ready for review)

File: README.md (hunk 2 above)
Comment (CodyCBakerPhD) [inline README.md]:
```suggestion
* `milliseconds_from_event_to_first_bin`: The time in milliseconds from the event to the first bin. A negative value indicates that the first bin is before the event whereas a positive value indicates that the first bin is after the event. 
```

**Comment by h-mayorquin** [comment]
Thanks, this is ready for review.

I added the timestamps description in the diagram and move the general description before the first code example. Plus, some wording (imo) improvements.

**Review by CodyCBakerPhD — APPROVED** [review]

**Pull request merged**